ebib-utils.el

```diff
@@ -305,6 +314,16 @@ not change the default sort."
                        (choice (const :tag "Ascending Sort" ascend)
                                (const :tag "Descending Sort" descend)))))
 
+(defcustom ebib-field-sort-functions-alist '(("Chapter" . ebib-compare-numerical-strings)
+					     ("Sortyear" . ebib-compare-numerical-strings)
+					     ("Volume" . ebib-compare-numerical-strings)
+					     ("Volumes" . ebib-compare-numerical-strings))
+  "Alist of bib(la)tex field names and corresponding functions used
+for sorting them in the index.
```

Comment:
I'd prefer it if the first line of the doc string is a whole sentence. (This is recommended for functions. Not sure if it's also recommended for variables / defcustoms, but I prefer to stick to it.)

Reply:
Good to know. Done.